command/auth_list.go

```diff
@@ -143,7 +143,7 @@ func (c *AuthListCommand) detailedMounts(auths map[string]*api.AuthMount) []stri
 		}
 	}
 
-	out := []string{"Path | Type | Accessor | Plugin | Default TTL | Max TTL | Token Type | Replication | Seal Wrap | Options | Description"}
+	out := []string{"Path | Plugin | Accessor | Default TTL | Max TTL | Token Type | Replication | Seal Wrap | Options | Description"}
```

Comment:
Should it be "Plugin" or "Type"?

Reply:
Brian had commented `s/Type/Plugin/`. I can go the other way if that's desired, but we should all agree.

Reply:
I think it's fine to just say "Plugin" now since everything is a plugin

Reply:
👍It might look weird listing older servers since that's not the case, but probably not a big deal.